internal/net/grpc/logger/logger_test.go

```diff
@@ -29,41 +33,54 @@ func TestInit(t *testing.T) {
 		name       string
 		want       want
 		checkFunc  func(want) error
-		beforeFunc func()
+		beforeFunc func(*testing.T)
 		afterFunc  func()
 	}
 	defaultCheckFunc := func(w want) error {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "set logger success with verbosity level is not set",
+			checkFunc: func(w want) error {
+				if grpclog.V(1) {
+					return errors.New("verbosity level is set")
+				}
+				return nil
+			},
+			afterFunc: func() {
+				once = sync.Once{}
+			},
+		},
+		{
+			name: "set logger success with verbosity level is set",
+			beforeFunc: func(t *testing.T) {
+				t.Setenv("GRPC_GO_LOG_VERBOSITY_LEVEL", "2")
+			},
+			checkFunc: func(w want) error {
+				if !grpclog.V(1) {
+					return errors.New("verbosity level 1 is not set")
+				}
+				if !grpclog.V(2) {
+					return errors.New("verbosity level is not set")
+				}
+				if grpclog.V(3) {
+					return errors.New("verbosity level is not correct")
+				}
+				return nil
+			},
+			afterFunc: func() {
+				once = sync.Once{}
+			},
+		},
```

Comment:
`want` is not set :thinking:

Reply:
the expected behavior is checked in the checkfunc, and this function returns nothing, so it is not set. 🤔

Reply:
So, why you did not change `checkFunc`?

Reply:
it should be `checkFunc()` not `checkFunc(w want)`

Reply:
fixed